app/assets/javascripts/discourse/app/templates/components/share-popup.hbs

```diff
@@ -34,6 +34,11 @@
       </div>
     {{/if}}
 
-    {{d-button action="close" class="btn btn-flat close" icon="times" aria-label="share.close" title="share.close"}}
+    {{d-button action=(action "close")
```

Comment:
```suggestion
    {{d-button
      action=(action "close")
```

Reply:
yes agreed, I didn't want to nitpick, but 😅 

Hope we can have auto formatting of templates soon.